docassemble/ALAutomatedTestingTests/al_set_up_testing.py

```diff
@@ -0,0 +1,48 @@
+from github import Github  # PyGithub
+import requests
+from nacl import encoding, public  # pynacl
+# TODO: Reduce to just one encryption library :/
+import codecs
+from base64 import b64encode
+import re
+import json
+from docassemble.base.util import log
+from docassemble.base.core import DAObject
+
+# reference:
+# Mostly: https://pygithub.readthedocs.io/en/latest/introduction.html
+
+class TestInstaller(DAObject):
+  def init( self, *pargs, **kwargs ):
+    super().init(*pargs, **kwargs)
+  
+  def set_da_info( self ):
+    """Use the interview url to get the user's Playground id."""
+    # Can we get granular with error messages?
+    log( 1, 'console' )
```

Comment:
Are these debugging statements meant to be in the final commit? If so, it's probably worth replacing `1` with some text like `started function set_da_info` etc.

Reply:
They are definitely not, thanks!